readthedocs/restapi/serializers.py

```diff
@@ -41,17 +41,24 @@ class Meta:
 
 
 class BuildSerializer(serializers.ModelSerializer):
+
     """Readonly version of the build serializer, used for user facing display"""
 
     commands = BuildCommandSerializer(many=True, read_only=True)
     state_display = serializers.ReadOnlyField(source='get_state_display')
 
+
+class VersionFullSerializer(VersionSerializer):
+
+    '''Serializer for all fields on version model'''
+
```

Comment:
Not sure what has happened here, `BuildSerializer` above lost its `Meta` inner class. Seems this should be the `Version` model as well?